docs/format_examples_datastore.py

```diff
@@ -0,0 +1,204 @@
+import pooch
+from pathlib import Path
+
+EXAMPLES_FOLDER = Path('format_examples')
+
+format_examples = [
+    dict(
+        name='TIFF (Tagged Image File Format)',
+        reference='https://www.itu.int/itudoc/itu-t/com16/tiff-fx/docs/tiff6.pdf',
+        examples=[
+            dict(
+                filename='A5.pattern3.tif',
+                url='https://data.kitware.com/api/v1/item/66e1af78560e61279679128a/download',
+                hash='c6d198c3e0a1c46bd44f2a1c2c594db9c91a013e98a127d7eed727cd1b0d8d1e',
+            ),
+            dict(
+                filename='sample_image.ptif',
+                url='https://data.kitware.com/api/v1/file/5be43e398d777f217991e21f/download',
+                hash='d73cbf87aac8f9cd9544b20d6c42537f4c609da15817928c8824d57049a2749a',
+            ),
+        ],
+    ),
+    dict(
+        name='Zarr',
+        reference='https://wiki.earthdata.nasa.gov/display/ESO/Zarr+Format',
+        examples=[
+            dict(
+                filename='WD1.1_17-03_WT_MP.ome.zarr.zip',
+                url='https://data.kitware.com/api/v1/file/6544e995fdc1508d8bec1838/download',
+                hash='f7dfb6683107d1cb21a360e31752b02641a16e0d32a87698669838418f108831',
+            ),
+            dict(
+                filename='normmedia_8well_col2_livecellgfp.db',
+                url='https://data.kitware.com/api/v1/file/6544e9f5fdc1508d8bec1860/download',
+                hash='7d6cfb6967f79eb5201eebbe93418e26445676d9ea95a3bf49ff4ffbf0993e39',
+            ),
+        ],
+    ),
+    dict(
+        name='PNG (Portable Network Graphics)',
+        reference='http://www.libpng.org/pub/png/spec/1.2/PNG-Structure.html',
+        examples=[
+            dict(
+                filename='Animated_PNG_example_bouncing_beach_ball.png',
+                url='https://upload.wikimedia.org/wikipedia/commons/1/14/Animated_PNG_example_bouncing_beach_ball.png',
+                hash='3b28e2462f1b31d0d15d795e6e58baf397899c3f864be7034bf47939b5bbbc3b',
+            ),
+        ],
+    ),
+    dict(
+        name='JPEG (Joint Photographic Experts Group)',
+        reference='https://jpeg.org/jpeg/',
+        examples=[
+            dict(
+                filename='albania.jpg',
+                url='https://github.com/OSGeo/gdal/blob/master/autotest/gdrivers/data/jpeg/albania.jpg',
+            ),
+            dict(
+                filename='clouds.jpeg',
+                url='https://data.kitware.com/api/v1/file/5afd936c8d777f15ebe1b4d4/download',
+                hash='fb195bb9dae13aa85d9f46f86ab9441a3b436dabbf57c68149453cba691d4391',
+            ),
+        ],
+    ),
+    dict(
+        name='JPEG (Joint Photographic Experts Group) 2000',
```

Comment:
I would reorder this "JPEG 2000 (Joint ...)", rather than explain the acronym in the middle of the format name.